2-mandatory/1-create-functions.js

```diff
@@ -33,7 +41,8 @@ Write a function that:
 - Returns a new array containing the same elements, but without the element at the passed index.
 */
 
-function remove() {
+function remove(array,index) {
+  return array.slice(0,index).concat(array.slice(index+1));
 }
```

Comment:
Correct. You got the "index + 1" part right meaning "the rest of the array". I personally prefer the spread operator:
```javascript
return [ ...arr.slice(0, index), ...arr.slice(index + 1) ];
```